src/Validator/Constraints/ExcludedMemberValidator.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Validator\Constraints;
+
+use App\Entity\ExcludeModification;
+use App\Entity\Member;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class ExcludedMemberValidator extends ConstraintValidator
+{
+
+    public function validate($value, Constraint $constraint): void
```

Comment:
### 🤓 Nitpick: `$value` is a bad name.

Maybe some `$excludeMod` will be better to understand